tests/unit/orders-update.spec.js

```diff
@@ -243,3 +251,64 @@ test(`buildUpdateActions
 
   t.end()
 })
+
+test(`getStateReferences
+  should fetch state and return reference type and state id`, (t) => {
+  const updater = newOrdersUpdate()
+  const mockData = { id: '53 65 6c 77 79 6e' }
+  const mockResult = Promise.resolve({
+    body: {
+      total: 1,
+      results: [mockData],
+    },
+  })
+
+  sinon.stub(updater.client.states, 'where', () => ({
+    fetch: () => mockResult,
+  }))
+
+  updater.getStateReference('testState').then((result) => {
+    t.ok(result)
+    t.equal(
+      result.typeId,
+      'state',
+      'reference type \'state\' is added to result',
+    )
+    t.ok(result.id, 'State Id is added to result')
+    t.end()
+  })
+})
+
+test(`getStateReferences
+  should return if no result is returned`, (t) => {
+  const updater = newOrdersUpdate()
+  const mockResult = Promise.resolve({
+    body: {
+      total: 0,
+      count: 0,
+      results: [],
+    },
+  })
+
+  sinon.stub(updater.client.states, 'where', () => ({
+    fetch: () => mockResult,
+  }))
+  updater.getStateReference('testState').then((result) => {
+    t.ok(result)
+    t.equal(
+      result.typeId,
+      'state',
+      'reference type \'state\' is added to result',
+    )
+    t.ok(result.id, 'State Id is added to result')
+    t.end()
+  }).catch((error) => {
+    t.ok(error)
```

Comment:
Just removed the `t.ok` checks since the test will fail any way if the data does not exist.